ginga/table/AstroTable.py

```diff
@@ -23,7 +23,7 @@ class AstroTableHeader(Header):
     pass
 
 
-class AstroTable(Callback.Callbacks):
+class AstroTable(ViewerObjectBase):
```

Comment:
I think `BaseImage` needs to be reserved for things that are fundamentally image like.  But now that we have two kinds of viewers it stands to reason that a "viewable base class" should exist.